Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM        ubuntu:16.04
+FROM        python:3.5
+ADD         requirements.txt /app/
+WORKDIR     /app
+RUN         apt-get update & apt-get install gcc -y
+RUN         pip install -r /app/requirements.txt
+ADD         . /app
+EXPOSE      80
+ENV         FLASK_APP "manage.py"
+ENV         FLASK_CONFIG "production"
+RUN         ["flask", "db", "init"]
```

Comment:
Will need to change this as soon as we start adding a `migrations` to the project.